### DISCUSSION TITLE
biome crate

### DISCUSSION THREAD

**Discussion by jeffparsons** [biomejs/biome#922] [General]
I have the "biome" crate on crates.io, but the "biome" I was making has since been absorbed into another project — so I don't need the name!

Can I transfer ownership of the name to somebody in this org?

**Comment by ematipico** [comment]
That would be awesome @jeffparsons ! You can transfer the ownership to me: https://crates.io/users/ematipico

**Comment by jeffparsons** [comment]
@ematipico Okay, I think I've done what I need to at my end. I'll leave it to you to remove me; I'm not sure what happens if a crate temporarily has no owner (if that's even possible — the UI makes it look like it is).

**Reply by ematipico** [reply]
Done! Thank you again 🎉
